specs/selectors/index.md

```diff
@@ -182,13 +182,21 @@ type ExploreConditional struct {
 	next Selector (rename ">")
 }
 
-## ExploreInterpretAs is a transformation that attempts to 'reify' the current node
-## using an ADL specified by 'as'. ADLs are recognized by agreed-upon strings,
-## similar to libp2p protocols.
-## The ExploreInterpretAs reification process may introduce a data-dependant amount
-## of budget on evaluation based on the specific traversal and ADL implementation.
-type ExploreInterpretAs struct {
-	as String (rename "c")
+## InterpretAs is a clause that instructs the traversal to attempt to 'reify' the current node
+## using an ADL, which is specified by the 'as' field.
```

Comment:
One thing missing from here is what it means for an ADL to be specified, what is this magic string? Not a critical change now, but we may want to get into that kind of detail at some point in our docs rather than hand-waving it away as a turtles-all-the-way-down concerns.